tests/expectations.json

```diff
@@ -3031,6 +2966,8 @@
   "built-ins/JSON/stringify/replacer-array-wrong-type.js": "CRASH",
   "built-ins/JSON/stringify/replacer-function-abrupt.js": "FAIL",
   "built-ins/JSON/stringify/replacer-function-arguments.js": "FAIL",
+  "built-ins/JSON/stringify/replacer-function-array-circular.js": "CRASH",
+  "built-ins/JSON/stringify/replacer-function-object-circular.js": "CRASH",
```

Comment:
Have you looked into why these crashes are happening?

Reply:
It's because I added the `stringify` builtin string and corrected the function name I think so it's to be expected.

Reply:
So we're talking tests that expect a `TypeError`, and were seeing one because the `stringify` function didn't exist, but now that it exists and they're calling it, they crash instead?